docs/functions/badge.ts

```diff
@@ -41,7 +41,7 @@ export default async (req: Request, context: Context) => {
     const style = query.get("style");
 
     // Build dynamic badge endpoint URL using current host
-    const base = `https://${host}/badge-endpoint/config/${tag}`;
+    const base = `https://${host}/badge/config/${tag}`;
```

Comment:
hm? I had been using https://openpolicyagent.org/badge/v1.5.0 now, sans /config/` and it works... should we perhaps use that simpler path? I'm not entirely certain that I understand what's going on here 😅

Reply:
The config path is used in the redirect and contains the format for the shields.